### ISSUE TITLE
Username property

### ISSUE THREAD

**Issue by pman12826** [neo4j/neo4j-dotnet-driver#234]
We are writing custom APIs using the Kerberos plugin.  Neo4j does a great job tagging the person doing the work and can be seen using dbms.listQueries, but sometimes I want to capture that username and pass to other methods in my c# code.  

Can the driver team add a method that returns the username.  For example:
IAuthToken token = AuthTokens.Kerberos(sBearer);
driver = GraphDatabase.Driver(boltAddress, token);
return driver.getUsername()
AND/OR
ISession session = driver.Session()
return session.getUsername()

In other words, I would like to capture the username out of the Kerberos ticket like Neo4j sees it in listQueries.

![listquery](https://user-images.githubusercontent.com/25668613/30444804-286a26b8-9952-11e7-9719-26e26d5feb13.jpg)

**Comment by ali-ince** [comment]
Hi @pman12826,

We currently do not plan to add `username` property that will be exposed by the driver API. However, what you've requested can be easily achieved by using `dbms.security.showCurrentUser` stored procedure. 

Thanks,

Ali
